src/Backend/JSON.idr

```diff
@@ -117,16 +130,17 @@ makeSchema schema defs = JObject
                   , ("properties", JObject [ ("value", JObject [("oneOf", JArray $ NEList.toList schema)] ) ])
                   ]
 
-generateSchema : TNamed 0 -> JSON
-generateSchema tn = makeSchema (singleton $ makeSubSchema' tn) (evalState (makeDefs' tn) [])
+-- generateSchema : TNamed 0 -> JSON
+-- generateSchema tn = makeSchema (singleton $ makeSubSchema' tn) (runMakeDefM (makeDefs' tn))
```

Comment:
Is this not working?

Reply:
It's never used so I intended to remove it but forgot